src/components/timeline/Markers.js

```diff
@@ -475,3 +475,21 @@ export const TimelineMarkersMemory = explicitConnect<
   },
   component: TimelineMarkers,
 });
+
+export const TimelineMarkersIPC = explicitConnect<OwnProps, StateProps, {||}>({
```

Comment:
Generally we try to add a bit of documentation for each component to provide a bit of context.